nio/store/database.py

```diff
@@ -397,6 +397,13 @@ def save_encrypted_rooms(self, rooms):
                 LegacyEncryptedRooms.account
             ]).on_conflict_ignore().execute()
 
+    @use_database_atomic
+    def delete_encrypted_room(self, room):
+        """Delete the room id for this account."""
+        LegacyEncryptedRooms.get(
```

Comment:
The legacy store shouldn't get new features, it's only there to provide a migration path to the new store.

Reply:
My bad, I didn't notice there were two stores. I moved the method and associed test to the new one.